hack/minikube-rook-setup.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+set -x
+set -e -o pipefail
+
+## Variables
+PROFILE="${PROFILE:-hub}"
+POOL_NAME="minikube"
+
+IMAGE_DIR="${IMAGE_DIR:-"/var/lib/libvirt/images"}"
+IMAGE_NAME="osd0"
+
+ROOK_SRC="${ROOK_SRC:-"https://raw.githubusercontent.com/rook/rook/release-1.6/cluster/examples/kubernetes/ceph/"}"
+
+## Usage
+usage()
+{
+	set +x
+	echo "Usage:"
+	echo "  $0 [create|start|stop|delete]"
```

Comment:
(nit) Curly braces are typically used instead to indicate something is required and square brackets, optional.  Though perhaps a subcommand is intended to be optional and `create` the default.